x-pack/plugins/security_solution/cypress/tasks/risk_scores/indices.ts

```diff
@@ -48,49 +42,30 @@ export const getCreateLegacyRiskScoreIndicesOptions = ({
       },
     },
   };
+};
+
+export const getLegacyRiskScoreIndicesOptions = ({
+  spaceId = 'default',
+  riskScoreEntity,
+}: {
+  spaceId?: string;
+  riskScoreEntity: RiskScoreEntity;
+}) => {
   return {
     index: getPivotTransformIndex(riskScoreEntity, spaceId),
-    mappings,
+    mappings: getRiskScoreMapping(riskScoreEntity),
   };
 };
 
-export const getCreateLegacyRiskScoreLatestIndicesOptions = ({
+export const getLegacyRiskScoreLatestIndicesOptions = ({
   spaceId = 'default',
   riskScoreEntity,
 }: {
   spaceId?: string;
   riskScoreEntity: RiskScoreEntity;
 }) => {
-  const mappings = {
-    properties: {
-      [`${riskScoreEntity}.name`]: {
-        type: 'keyword',
-      },
-      '@timestamp': {
-        type: 'date',
-      },
-      ingest_timestamp: {
-        type: 'date',
-      },
-      risk: {
-        type: 'text',
-        fields: {
-          keyword: {
-            type: 'keyword',
-          },
-        },
-      },
-      risk_stats: {
-        properties: {
-          risk_score: {
-            type: 'float',
-          },
-        },
-      },
-    },
-  };
   return {
     index: getLatestTransformIndex(riskScoreEntity, spaceId),
-    mappings,
+    mappings: getRiskScoreMapping(riskScoreEntity),
```

Comment:
Thanks for the clean up @MadameSheema ! , could you please rename `getRiskScoreMapping` to `getLegacyRiskScoreMapping`

Reply:
Sure!! Thanks @angorayc :)